src/Uno.UI/UI/Xaml/FontFamily.cs

```diff
@@ -11,9 +12,13 @@ public partial class FontFamily
 		public FontFamily(string familyName)
 		{
 			Source = familyName;
-
 			Init(familyName);
 
+			if (familyName.Equals("Segoe Fluent Icons,Segoe MDL2 Assets", StringComparison.OrdinalIgnoreCase))
+			{
+				Source = "Segoe MDL2 Assets";
+			}
```

Comment:
We probably should be supporting the fallback feature properly instead (comma separated fonts). We can keep this workaround for now to unblock the original code, but we need to track it as a missing feature in a new issue.